catalog/registry/catalog-registry-cswinputtransformer/src/main/java/ddf/catalog/registry/converter/RegistryPackageConverter.java

```diff
@@ -0,0 +1,667 @@
+
+                if (anyValue.isSetContent()) {
+
+                    for (Object content : anyValue.getContent()) {
+                        if (content instanceof JAXBElement) {
+
+                            JAXBElement jaxbElement = (JAXBElement) content;
+
+                            AbstractGeometryType geometry =
+                                    (AbstractGeometryType) jaxbElement.getValue();
+
+                            String convertedGeometry = getWKTFromGeometry(geometry, jaxbElement);
+
+                            if (StringUtils.isNotBlank(convertedGeometry)) {
+                                metacard.setAttribute(metacardAttributeName, convertedGeometry);
+                            }
+                        }
+                    }
+                }
+            }
+
+        }
+    }
+
+    private static Map<String, SlotType1> getSlotMap(List<SlotType1> slots) {
+        Map<String, SlotType1> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            slotMap.put(slot.getName(), slot);
+        }
+        return slotMap;
+    }
+
+    private static Map<String, List<SlotType1>> getSlotMapWithMultipleValues(
+            List<SlotType1> slots) {
+        Map<String, List<SlotType1>> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            if (!slotMap.containsKey(slot.getName())) {
+                List<SlotType1> slotList = new ArrayList<>();
+                slotList.add(slot);
+
+                slotMap.put(slot.getName(), slotList);
+            } else {
+                slotMap.get(slot.getName())
+                        .add(slot);
+            }
+        }
+        return slotMap;
+    }
+
+    private static String getStringFromIST(InternationalStringType internationalString) {
+        String stringValue = "";
+        List<LocalizedStringType> localizedStrings = internationalString.getLocalizedString();
+        if (CollectionUtils.isNotEmpty(localizedStrings)) {
+            LocalizedStringType localizedString = localizedStrings.get(0);
+            if (localizedString != null) {
+                stringValue = localizedString.getValue();
+            }
+        }
+
+        return stringValue;
+    }
+
+    private static void unsetMetacardAttribute(String metacardAttribute, RegistryMetacardImpl metacard) {
+        if (StringUtils.isNotBlank(metacardAttribute)) {
+            metacard.setAttribute(metacardAttribute, null);
+        }
+    }
+
+    private static void validateOrganization(OrganizationType organization)
+            throws RegistryConversionException {
+        if (StringUtils.isBlank(organization.getId())) {
+            String message =
+                    "Error converting registry object to metacard. Organization must have an ID set.";
+            throw new RegistryConversionException(message);
+        }
+    }
+
+    private static void validatePerson(PersonType person) throws RegistryConversionException {
```

Comment:
all the validates could be condensed into one method that takes in an Identifiable as an argument.